### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by matkt**
Signed-off-by: Karim TAAM <karim.t2am@gmail.com>

## PR Description

update dependencies

## Fixed Issue(s)
update log4j

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 21.12.0
+
+### Additions and Improvements
+* Update dependencies
+
+### Bug Fixes
+* Update log4j to 2.167.0.
```
Comment (fab-10) [inline CHANGELOG.md:9]: should be 2.17.0

**Review by fab-10 — CHANGES_REQUESTED** [review]
LGTM, just the fix in the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (matkt) [reply CHANGELOG.md:9]: thanks

Comment (matkt) [reply CHANGELOG.md]: done

**Review by fab-10 — APPROVED** [review]

**Pull request merged**